### PR TITLE
basic endpoint setup

### PR DESCRIPTION
**Pull request by zoeabryant**
- [x] an endpoint that invokes the function
- [x] some form of validation, like a required secret key
- [ ] prismic webhooks configured

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,2 +1,3 @@
 BUCKET_NAME_PREFIX="my-website-honestly"
 BUCKET_NAME_SUFFIX="dev"
+API_KEY='my-website-honestly'
```
Comment (lpil) [inline .env.example]: This should be very clear that it needs to be changed!

Comment (lpil) [inline .env.example]: Ah, misunderstanding. It seems this is the name of the key, AWS handles generation for us.

No need for this to be an env var then,

Comment (lpil) [inline .env.example]: Also needs to be named after the client that uses it :)

Comment (zoeabryant) [inline .env.example]: have clarified on slack that this is for the API_KEY name and definitely not the key itself. Going to move it inline as it's not a secret.

File: publish-service/index.js [hunk 2]
```diff
@@ -15,7 +16,11 @@ function doPublish(cb) {
 
 export function publish(event, context, cb) {
   try {
-    doPublish(cb);
+    if (event && event.body && event.body.secret === secretAuth) {
```
Comment (lpil) [inline publish-service/index.js]: Do we need to check for event?

Comment (zoeabryant) [inline publish-service/index.js]: I was conscious of 'cannot get body of undefined' errors - but I think it's highly unlikely to get undefined. You can get an empty body but probably not an actual undefined.
Will remove it.

**Comment by lpil** [comment]
What's going on here Zoe?

**Comment by asavin** [comment]
Closing this one as it is stale.

**Pull request closed without merging**